src/Filters/stream_filt.jl

```diff
@@ -58,7 +58,7 @@ end
 # Rational resampler FIR kernel
 mutable struct FIRRational{T}  <: FIRKernel{T}
     pfb::PFB{T}
-    ratio::Rational{Int}
```

Comment:
Moving away from a concrete type here might have a performance impact, though I haven't checked where `ratio` is used/how large the impact is. Alternatively, we could leave this as `Rational{Int}` and convert upon construction, no?